frontend/src/lib/components/details/person_media.svelte

```diff
@@ -44,31 +58,16 @@
 
 {#if media.length}
   <h1 class="text-center text-3xl pt-5">Starred in</h1>
-  <div
-    class="grid grid-cols-2 2xl:grid-cols-8 xl:grid-cols-7 lg:grid-cols-6 md:grid-cols-5 md:grid-cols-4 gap-3 p-2 pt-4"
-  >
-    {#each media.slice(0, currentMediaAmount) as media}
-      <a
-        in:fade
-        href={mediaIdToUrlConverter(media.id)}
-        data-sveltekit-prefetch
-        class="card compact bordered shadow-md
-                       hover:contrast-75 hover:ring-2 ring-primary"
-      >
-        <figure class="rounded-box">
-          <img src="{IMG_W342}{media.poster_path}" alt={getMediaTitle(media)} />
-        </figure>
-        {#if media.id.charAt(0) == "t"}
-          <div class="absolute top-0 right-0 mx-1 -mt-1 opacity-85">
-            <div class="badge badge-sm">TV</div>
-          </div>
-        {/if}
-      </a>
-    {/each}
-  </div>
-  {#if media.length > currentMediaAmount}
-    <InfiniteLoading on:infinite={loadMoreData} />
-  {:else}
-    <p class="text-center italic">Showing {media.length} results</p>
-  {/if}
+  {#await refreshLookupMedia()}
+    <Spinner timeout={false} />
```

Comment:
This is fine for now, but should we make an issue about the skeleton cards?

Reply:
#385